scripts/world_bank/health_nutrition_population/population_get_mcf.py

```diff
@@ -0,0 +1,35 @@
+def get_mcf(series_lst):
+    '''
+    gets mcf by splitting description and using fstrings
+    '''
+    nodes = []
+    #don't  use df; automate
+    with open("output\World_bank_hnp_population.mcf", 'w', encoding = 'utf-8') as file:
```

Comment:
please use os.path.join -- most of us are on unix environments, so the path separator is `/` not `\`. os.path.join will take care of that.